allocation.md

```diff
@@ -73,13 +73,23 @@ cpuCoreRequestAverage | Average number of CPU cores requested while running.
 cpuCoreUsageAverage | Average number of CPU cores used while running.
 cpuCoreHours | Cumulative CPU core-hours allocated.
 cpuCost | Cumulative cost of allocated CPU core-hours.
+cpuCostAdjustment | Matches CPU costs to reconciled assets and divides cost
 cpuEfficiency | Ratio of `cpuCoreUsageAverage`-to-`cpuCoreRequestAverage`, meant to represent the fraction of requested resources that were used.
+gpuCount | Number of GPUs allocated to the workload.
 gpuHours | Cumulative GPU-hours allocated.
 gpuCost | Cumulative cost of allocated GPU-hours.
+gpuCostAdjustment | Matches GPU costs to reconciled assets and divides cost
+networkTransferBytes | Cost of data being sent outside the network
+networkReceiveBytes | Cost of data being received by the network
```

Comment:
```suggestion
networkTransferBytes | Cost of network data being sent
networkReceiveBytes | Cost of network data being received
```

I'm pretty sure this is better wording, but please override if someone told you otherwise!